### ISSUE TITLE
php parsing issue

### ISSUE THREAD

**Issue by jcberleur** [facebook/pfff#127]
I get "warning: parsing problem in FILE" when trying it on a file with parent in a namespace while working in php/hhvm

```php
<?php

namespace foo\parent\bar {
    class Foo { }
}

namespace {
    $foo = new foo\parent\bar\Foo();
    var_dump($foo);
}
```

**Comment by aryx** [comment]
Yes it's because parent is considered a keyword. IMHO you should not use namespace names
which happen to also be  keywords. I know the official PHP languages allows that, but I'm not
sure it's a good idea.
